var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -676,9 +674,9 @@ def suitesparse_components(self):
     @property
     def sundials_components(self):
         """Return the SUNDIALS components needed by MFEM."""
-        sun_comps = 'arkode,cvode,nvecserial,kinsol'
+        sun_comps = 'arkode,cvodes,nvecserial,kinsol'
```

Comment:
For building older versions of mfem with any version sundials: I think `cvodes` provides the same interface as `cvode` (with additions), so this should be OK -- @gardner48 correct me if I'm wrong.

Reply:
Yes, using `cvodes` should work find.